### DISCUSSION TITLE
Pretrained weights

### DISCUSSION THREAD

**Discussion by maloq** [microsoft/ClimaX#8] [General]
Thank you for sourcing your code. Are model weights publicly available?

**Comment by rejuvyesh** [comment]
They are! See example in https://microsoft.github.io/ClimaX/usage/ .

Links:
- https://climaxrelease.blob.core.windows.net/checkpoints/ClimaX-5.625deg.ckpt
- https://climaxrelease.blob.core.windows.net/checkpoints/ClimaX-1.40625deg.ckpt

**Reply by kayden-aks** [reply]
Hi all, 

I encountered an issue while attempting to access the pre-trained model through the provided link. Upon request, I received a response code 409 with the message: "Public access is not permitted on this storage account." This suggests that the current configuration does not allow public access to the storage account. Can you recheck the link?

<img width="1167" alt="Screenshot 2023-06-12 at 16 26 06" src="https://github.com/microsoft/ClimaX/assets/32960645/9d81d3a2-72c7-4d08-a647-2e96fd408bf4">

**Reply by rejuvyesh** [reply]
Apologies for the delay. Certain storage policies changed in our Azure bucket and we are working on updated links.

**Reply by atiyeh-ashari** [reply]
Hi there,
Are there any updates on the permission problem or any idea when it will be resolved?

**Reply by rejuvyesh** [reply]
Oh, the docs have already been updated to point to the new locations! Please see https://huggingface.co/datasets/tungnd/climax_ckpts/
